### DISCUSSION TITLE
it does nothing

### DISCUSSION THREAD

**Discussion by hhp-the-maestro** [codingjoe/ssdp#69] [Q&A]
I have come across many resource online still none works including this. just runs without errors , no output. if someone helps me out of this it would be great.

**Comment by codingjoe** [comment]
Hi @hhp-the-maestro,

Thank you for reaching out. Would you be so kind as to share a little more information about what you're trying to achieve and what you've tried thus far? I am afraid, right now, I don't have much to go on to help you out.

Cheers,
Joe
